src/KM_Music.pas

```diff
@@ -264,12 +266,20 @@ procedure TMusicLib.ScanMusicTracks(aPath: UnicodeString);
 
 
 procedure TMusicLib.PlayMenuTrack;
+var
+  prevVolume: Single;
 begin
   if not IsMusicInitialized then Exit;
   if fMusicCount = 0 then Exit; //no music files found
   if fMusicIndex = 0 then Exit; //It's already playing
   fMusicIndex := 0;
-  PlayMusicFile(fMusicTracks[0]);
+  // There was audio crackling after loading screen, here we fix it
+  prevVolume := MusicGain; // Store volume setting to re-apply later on
```

Comment:
Comments on this and following lines are superfluous and can be removed - they repeat the code